src/app/shared/components/select/select.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="container pb-2 pt-2">
     <label *ngIf="label">{{ label }}</label>
     <div class="select">
-        <div class="display-area">
+        <div class="display-area" (click)="toggleIsShowingOptions()">
             <div class="display-text">
                 {{ !!selectedOption ? selectedOption.name : (placeholder ?? label) }}
             </div>
         </div>
 
-        <div class="select-dropdown" >
+        <div class="select-dropdown" [class.block]="isShowingOptions">
```

Comment:
Instead of toggling a class, you can use `*ngIf="isShowingOptions"`. That way, `display: none;` can be removed from the `.select-dropdown` selector.

Reply:
Done